lib/generators/provider/templates/.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+bundler_args: --no-deployment
```

Comment:
> I want this to not exist, and would prefer we fix the double-bundle-update instead. See also #15503

That is fair, that is the other option, and that works as well.

> > bundling twice
> can you point me to where this happens?

@durandom If you expand the output of the `bin/setup`, you will see the `bundle install` output there.  That happens prior to the one that Travis gives by default.